### PR TITLE
Reduce minimum slides for autoplay to 2 on <amp-carousel>

### LINKED ISSUE #5164: amp-carousel autoplay not playing with 2 images
I am trying to implement an amp-carousel with autoplay. Yet it is not autoplaying.
I got the code from the examples.

Below is my code:

`<amp-carousel width="2000"
  height="403"
  layout="responsive"
  type="slides"
  autoplay
  delay="3500">
<amp-img src="http://www.webtondemo.nl/amptest/wp-content/uploads/WEBTON-belgie-01.jpg"
    width="2000"
    height="403"
    layout="responsive"></amp-img>
<amp-img src="http://www.webtondemo.nl/amptest/wp-content/uploads/WEBTON-belgie-02.jpg"
    width="2000"
    height="403"
    layout="responsive"></amp-img>
</amp-carousel>`

### PR DESCRIPTION
**Pull request by nainar**
Fixes #5164

### CODE REVIEW

File: extensions/amp-carousel/0.1/slidescroll.js [hunk 1]
```diff
@@ -208,7 +208,7 @@ export class AmpSlideScroll extends BaseSlides {
 
   /** @override */
   isLoopingEligible() {
-    return this.noOfSlides_ > 2;
+    return this.noOfSlides_ > 1;
```
Comment (cathyxz) [inline extensions/amp-carousel/0.1/slidescroll.js]: Talked offline, lets verify that the `hasPrev` and `hasNext` functions here still behave correctly? =)

Comment (nainar) [reply extensions/amp-carousel/0.1/slidescroll.js]: Made a change to make sure that the slides appear in the correct order. Seems hacky to me. Let me know what you think?

File: extensions/amp-carousel/0.1/test/test-slidescroll.js [hunk 2]
```diff
@@ -686,6 +686,67 @@ describes.realWin('SlideScroll', {
       });
     });
 
+    it.only('show correct slides when looping with `autoplay` on for 2 slides', () => {
```
Comment (cathyxz) [inline extensions/amp-carousel/0.1/test/test-slidescroll.js]: Remove "only".

Comment (nainar) [reply extensions/amp-carousel/0.1/test/test-slidescroll.js]: Done.

File: extensions/amp-carousel/0.1/slidescroll.js (hunk 1 above)
Comment (nainar) [reply extensions/amp-carousel/0.1/slidescroll.js]: Made the condition easier to follow.

**Comment by nainar** [comment]
@cathyxz made the changes we discussed offline. PTAL?

**Comment by nainar** [comment]
@jridgewell - any idea why this change is causing bundlesize to increase? 
https://travis-ci.org/ampproject/amphtml/jobs/406048120#L617

Should I bump the limit by 0.01?

**Pull request merged**